doc/integrator/create_application.rst.tmpl

```diff
@@ -120,12 +120,15 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
 
+See also :ref:`integrator_structure` for more information.
+
+All the administrator and integrator guides indicate if the section doesn't concern the simple
```

Comment:
Suggestion:
"The documentation sections indicate whether the current section applies to the simple application mode or not."